src/Context/JsonContext.php

```diff
@@ -76,6 +80,28 @@ public function theJsonNodesShoudBeEqualTo(TableNode $nodes)
     }
 
     /**
+     * Checks, that given JSON node is not equal to given value
+     *
+     * @Then the JSON node :node should not be equal to :text
+     */
+    public function theJsonNodeShouldNotBeEqualTo($node, $text)
+    {
+        $json = $this->getJson();
```

Comment:
Could you factorise ? With the other sentence with a protected or private common function